published/024-2019-jan-8.md

```diff
@@ -0,0 +1,345 @@
++ lanzafame
++ laser
++ leekt216
++ leerspace
++ leo6104
++ LEonGAo1991
++ lexrus
++ lidel
++ LinusU
++ litzenberger
++ lgierth
++ luizirber
++ machawk1
++ magik6k
++ maksimyugai
++ manandbytes
++ marcooliveira
++ marpett
++ marten-seeman
++ matrushka
++ mburns
++ mcollina
++ meiqimichelle
++ mhchia
++ mishmosh
++ michaelavila
++ michaelmcandrew
++ MichaelMure
++ MidnightLightning
++ mikeal
++ miyazono
++ mgoelzer
++ mkg20001
++ momack2
++ Mr0grog
++ My9Bot
++ ncocchiaro
++ negamaxi
++ NeoTeo
++ nGoline
++ nicola
++ Nipol
++ nstrelow
++ oed
++ olebedev
++ olizilla
++ OR13
++ Otto-AA
++ overbool
++ parkan
++ PascalPrecht
++ passerby888
++ PedroMiguelSS
++ petethomas
++ pgte
++ phritz
++ pkafei
++ PlayerWithoutName
++ pmthomps
++ pors
++ postables
++ Powersource
++ preetsinojiya
++ prettymuchbryce
++ PunkChameleon
++ qpakzk
++ racin
++ raoulmillais
++ raulk
++ renrutnnej
++ requilence
++ RichardLitt
++ richardschneider
++ ridewindx
++ rjharmon
++ rjrbt
++ rob-deutsch
++ robbsolter
++ robotamer
++ rodolfo-r
++ Rokko11
++ romaric-juniet
++ rwv
++ s1na
++ sahib
++ sameer
++ samli88
++ satazor
++ satoru
++ sbani
++ schomatis
++ seungwon-kang
++ shunkino
++ skylarnorris
++ slandau3
++ soupdiver
++ Stebalien
++ steefmin
++ steveruckdashel
++ sublimino
++ suzaku
++ taigrr
++ tarekbadrshalaan
++ taylormike
++ terichadbourne
++ tdiesler
++ the-owl
++ thiagodelgardo111
++ thomasbeta
++ Tirz
++ travisperson
++ TroyWilson1
++ TUSF
++ ult-yole
++ Varunram
++ vasco-santos
++ victorb
++ vmx
++ vukovinski
++ vutsalsinghal
++ vyzo
++ wanderingstan
++ warpfork
++ waynewyong
++ webmaniak
++ wei-1
++ whereswaldon
++ whilei
++ whyrusleeping
++ wraithgar
++ ya7ya
++ zbigniewzolnierowicz
++ zcstarr
++ ZenGround0
+
+Huge thanks to all of you for your time, your expertise, and your contributions. We couldn’t do it without each and every one of you. ❤️
+
+# Building the web with Awesome IPFS
+
+These are just a few of the new apps and tools that were submitted to the [new Awesome IPFS](https://awesome.ipfs.io/) site, showing that people are excited about building things with IPFS. 
+
++ [brig](https://brig.readthedocs.io/en/latest) - File synchronization with git like interface and FUSE filesystem.
+
++ [IPFSStore](https://ipfsstore.it) - Pinning paid with Steem
+
++ [Autonomica “IPFS Social Proof”](https://github.com/IBM/ipfs-social-proof) - Autonomica is a Keybase-like Dapp for creating an identity and proving this identity via published social media and web proofs.
+
++ [TallyLab](https://tallylab.com/) - Local-first, end-to-end encrypted diary app for capturing, analyzing, and sharing data about any and everything.
+
++ [Pinata](https://pinata.cloud) - Build and manage your dapp through Pinata’s REST API and IPFS toolkit.
+
++ [IPFS Desktop](https://github.com/ipfs/ipfs-desktop) - Run your IPFS node on your machine without having to bother with command line tools. Manage your node, add your files, easily change the settings... everything from just one interface.
```

Comment:
@olizilla do we want to low key announce ipfs-desktop before having auto-updates sorted out? 
```suggestion
+
```
(I suggest we remove it from this list and keep it about external projects)